### DISCUSSION TITLE
GSoc 2023

### DISCUSSION THREAD

**Discussion by GajananPalepwad** [robolectric/robolectric#8070] [Q&A]
Dear oppia,

I'm Gajanan Palepwad, a second-year undergraduate student interested in All java and android projects in oppia for GSOC 2023. I have over 2 years of experience in Java and Android app development using Java, XML will enable me to make valuable contributions to your project. Additionally, I have developed an Android app that has received 700+ downloads on Google Play Store and I'm also familiar with some cloud technologies. Furthermore, I completed an internship with a high-tech startup that equipped me with the necessary skills to successfully complete the project.

I am writing to express my keen interest in the project. After reading the project description and requirements, I am confident that my skills and experience align with the project goals.

I am excited about the opportunity to work on this project with your organization, and I am committed to delivering high-quality work within the given timeframe. Furthermore, I am open to discussing any additional requirements or modifications to the project to ensure its success.

I have attached some important links and my portfolio for your reference. Please do not hesitate to contact me if you require any further information or to schedule an interview. Thank you for considering my application.

Mail: [[gajananpalepwad@gmail.com](mailto:gajananpalepwad@gmail.com)]
Linkedin: https://www.linkedin.com/in/gajanan-palepwad-094794214/

Best regards,
Gajanan Palepwad

**Comment by utzcoz** [comment]
Sorry, Robolectric will not participate in the GSoC 2023.
